src/transactions/feed/queryHelper.ts

```diff
@@ -60,70 +62,96 @@ const deduplicateTransactions = (
   return transactionsWithoutDuplicatedHash
 }
 
+export function getAllowedChains(): Array<Chain> {
+  return getFeatureGate(StatsigFeatureGates.SHOW_MULTI_CHAIN_TRANSFERS)
```

Comment:
seems like another way of doing this could be to make the value of the feature gate a list of chains

Reply:
it would give us more granular control over which chains are supported at the feature flag level

Reply:
Yeah, added to the dynamic config linked above, instead of a feature gate.